### ISSUE TITLE
[BUG] The HP Indicator is stuck and smushed in the top left corner, and there's no HP number.

### ISSUE THREAD

**Issue by xDroflLOL** [CriticalFlaw/TF2HUD.Fixes#197]
**Issue**
Title

**Specifications**
Additional information about the issue: Windows 10 21H2, game was running at 1920x1080, all I did was install this hud with these settings.
![image](https://github.com/CriticalFlaw/TF2HUD.Fixes/assets/87008513/d24d40f7-8569-4ebd-8de4-cf37bc2c16ef)

**Screenshots**
![image](https://github.com/CriticalFlaw/TF2HUD.Fixes/assets/87008513/c7b1255f-a1b4-475a-93d8-c500339424f9)

**Comment by Presto06** [comment]
yeah this happened to me too when i installed off the hud editor, downloading from github was fine though. maybe schema needs updating

**Comment by xDroflLOL** [comment]
> yeah this happened to me too when i installed off the hud editor, downloading from github was fine though. maybe schema needs updating

Hmmm...

**Comment by CriticalFlaw** [comment]
Moving this issue to the editor repo for tracking.
